assets/css/style.css

```diff
@@ -57,39 +68,276 @@ a#img_link {
   padding: 50px 30px 50px 30px;
 }
 
-#name_title {
+.title {
+  margin: 10px;
   font-weight: 700;
   font-size: 40px;
   line-height: 52px;
   color: var(--black-shade-title);
 }
 
-#subtitle {
+.subtitle {
   font-weight: 400;
   font-size: 16px;
   line-height: 24px;
   color: var(--black-shade-subtitle);
+  margin: 10px;
 }
 
-#mini_title {
+.mini_title {
   font-weight: 700;
   font-size: 16px;
   line-height: 44px;
   text-align: left;
-  color: var(--blue-shade-text);
+  color: var(--black-shade-title);
+  margin: 10px;
 }
 
-ul {
+.social_links ul {
+  display: flex;
   list-style: none;
-  margin: 0;
+  margin: 10px;
   padding: 0;
 }
 
-li {
-  display: inline;
+.social_links ul li {
+  display: flex;
   margin: -5px;
 }
 
 .icon_image {
   padding: 10px;
 }
+
+#works {
+  height: 100%;
+  background-color: var(--white-shade-1);
+  padding: 16px;
+  display: grid;
+  gap: 10px;
+  grid-template-rows: 100vh 100vh 100vh 100vh;
+  justify-content: space-around;
+}
+
+.card {
+  height: 98vh;
+  display: flex;
```

Comment:
❤️

Reply:
Thank you so much for your review. Just to confirm can I go ahead and merge?